src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java

```diff
@@ -0,0 +1,143 @@
+package org.folio.circulation.domain.policy.lostitem;
+
+import static org.folio.circulation.domain.policy.Period.from;
+import static org.joda.time.DateTime.now;
+import static org.joda.time.DateTimeZone.UTC;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.folio.circulation.domain.policy.Period;
+import org.joda.time.DateTime;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import api.support.builders.LostItemFeePolicyBuilder;
+import io.vertx.core.json.JsonObject;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+@RunWith(JUnitParamsRunner.class)
+public class LostItemPolicyTest {
+
+  @Test
+  @Parameters( {
+    "Minutes, 78",
+    "Hours, 9",
+    "Days, 66",
+    "Weeks, 23",
+    "Months, 13",
+  })
+  public void shouldNotAgeItemToLostIfDueDateAfterNow(String interval, int duration) {
+    final Period period = from(duration, interval);
+    final LostItemPolicy lostItemPolicy = lostItemPolicyWithAgePeriod(period);
+
+    assertFalse(lostItemPolicy.canAgeLoanToLost(now(UTC)));
+  }
+
+  @Test
+  @Parameters( {
+    "Minutes, 43",
+    "Hours, 12",
+    "Days, 29",
+    "Weeks, 1",
+    "Months, 5",
+  })
+  public void shouldAgeItemToLostIfDueDateBeforeNow(String interval, int duration) {
+    final Period period = from(duration, interval);
+    final LostItemPolicy lostItemPolicy = lostItemPolicyWithAgePeriod(period);
+
+    final DateTime loanDueDate = now(UTC).minus(period.timePeriod()).minusSeconds(1);
+
+    assertTrue(lostItemPolicy.canAgeLoanToLost(loanDueDate));
+  }
+
+  @Test
+  @Parameters( {
+    "Minutes, 123",
+    "Hours, 99",
+    "Days, 64",
+    "Weeks, 2",
+    "Months, 3",
+  })
+  public void shouldAgeItemToLostIfDueDateIsNow(String interval, int duration) {
```

Comment:
Similarly, wouldn't the item only age to lost when the due date is now if the age to lost period was of zero length?

Reply:
If the _age to lost period_ is zero length then age to lost is disabled. 

If _bill patron after age to lost_ is zero length it means that patron is billed immediately (in our design - on the next charge fees job run).

Reply:
Much like my comment above, I think using only the due date and now in the name suggests it is a comparison between them. 

Whereas what I believe the test is trying to demonstrate is `item should be aged to lost when due date is older than the ageing period`. Is that the case?

Reply:
It demonstrates that `dueDate + period == now()`.

Reply:
Thanks. Please can the name reflect that, rather than only the due date and now.